addons/survey/views/survey_user_views.xml

```diff
@@ -95,7 +95,7 @@
                                     <field name="page_id" optional="hidden"/>
                                     <field name="question_id"/>
                                     <field name="answer_type" optional="hidden"/>
-                                    <field name="skipped" hide="1"/>
+                                    <field name="skipped"/>
```

Comment:
Hello @mehjabinfarsana , 

Thanks for the PR.

Actually, thanks to
```xml
<list decoration-muted="skipped == True">
```
the answers that are skipped are in grey (and the answer column contains "Skipped") so that column is meant to be invisible, I think.
So instead, maybe it would be better to hide it like this:
```
<field name="skipped" column_invisible="True"/>
```
What do you think ?